text/src/autogluon/text/text_prediction/presets.py

```diff
@@ -0,0 +1,234 @@
+__all__ = ['ag_text_presets', 'list_presets']
+
+import copy
+import logging
+import functools
+from autogluon_contrib_nlp.utils.registry import Registry
+from autogluon.core import space
+
+logger = logging.getLogger(__name__)  # return root logger
+
+# TODO, Consider to move the registry to core
+ag_text_presets = Registry('ag_text_presets')
+
+
+def list_presets():
+    """List the presets available in AutoGluon-Text"""
+    return ag_text_presets.list_keys()
+
+
+def base() -> dict:
+    """The default hyperparameters.
+
+    It will have a version key and a list of candidate models.
+    Each model has its own search space inside.
+    """
+    ret = {
+        'models': {
+            'MultimodalTextModel': {
+                'backend': 'gluonnlp_v0',
+                'search_space': {
+                    'model.backbone.name': 'google_electra_base',
+                    'optimization.batch_size': 128,
+                    'optimization.per_device_batch_size': 8,
+                    'optimization.num_train_epochs': 10,
```

Comment:
Shouldn't `num_train_epochs` be increased for `'best_quality'` preset?
In the future, we should add a `early_stopping_patience_epochs` hyperparameter instead and allow `num_train_epochs` to be much larger by default.

Reply:
I believe the reasoning is it isn't easy to adjust the triangular learning rate, and having a high num_train_epochs would impact the triangular learning rate. Ideally post v0.1 we want to have LR cap off at a certain point and then stay constant to enable more epochs without impacting most training runs.

Reply:
Ideally we should actually set to some values like 50 for number of epochs but it may take really long for training some models.

Reply:
Let me change to train for 20 epochs for best quality?